static/scss/partials/landing/a.scss

```diff
@@ -63,3 +80,216 @@
         }
     }
 }
+
+
+
+// How it works section
+
+.how-it-works-content {
+    width: 100%;
+
+    .how-it-works-header {
+        @include text-title-sm;
+    }
+
+    .how-it-works-steps {
+        width: 100%;
+        margin-top: $layout-sm;
+        margin-bottom: $layout-lg;
+
+        &.mzp-l-columns {
+            margin-top: $layout-md;
+
+            .step-illustration {
+                left: 50%;
+                transform: translateX(-50%);
+                position: relative;
```

Comment:
While this works, messing with positioning can get complex pretty quickly (resulting in new [stacking contexts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context), affecting the location of absolutely positioned elements, and probably more), so unless it's absolutely necessary, I'd recommend avoiding it if possible.

And in this case, I don't think it's necessary: if you want to horizontally center something, you can either make it an inline element (which an `<img>` is by default) and wrap it in something with `text-align: center`, or make it a block-level element (`display: block`) and set its left and right margins to `auto`.

Likewise for `.step-text` below, although I think in that case just removing the three lines doesn't change anything?